packages/cli/package.json

```diff
@@ -3,7 +3,8 @@
   "type": "module",
   "private": true,
   "bin": {
-    "dev-web": "./src/dev.mjs"
+    "dev-web": "./src/dev.mjs",
+    "dev-web:local": "./src/dev-local.mjs"
```

Comment:
Not that necessary, but it would be convient for someone (like me) to create a task for it to auto relaunch the application (as it does not need any interactive selection).

Reply:
I'd like to remove it? I think we could add instruct in the document instead here

Reply:
done.